modules/veteran/app/workers/veteran/base_reloader.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module Veteran
+  class BaseReloader
+    include Sidekiq::Worker
+    BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
+
+    private
+
+    def find_or_initialize(hash)
+      rep = Veteran::Service::Representative.find_or_initialize_by(representative_id: hash['Registration Num'],
+                                                                   first_name: hash['First Name'],
+                                                                   last_name: hash['Last Name'])
+      rep.poa_codes ||= []
```

Comment:
Instead of an OR Equals, we could just set this as a DB default, then we dont need initial the empty arrays

Reply:
Was trying to avoid a column change like this but I agree that it's the better approach.

Reply:
Change made to migration